[Uptime] [Test] API Test update

justinkambic opened the pull request:
## Summary

As part of https://github.com/elastic/kibana/pull/38895, we added new GraphQL API calls. This patch adds API tests for them to ensure they are returning expected data.

## Testing this PR
Simply running the API tests and ensuring the code adheres to the established patterns in this part of the codebase should be sufficient.

elasticmachine commented:
Pinging @elastic/uptime

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pull-request/5462/)

elasticmachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pull-request/5608/)

elasticmachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pull-request/6119/)

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pull-request/6333/)

justinkambic commented:
@jasonrhodes this PR is failing CI because of the issue we fixed yesterday with https://github.com/elastic/kibana/pull/40526. I rebased this this morning before merging that PR. I will rebase again and we should be good once more.

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pull-request/6376/)

jasonrhodes commented on x-pack/test/api_integration/apis/uptime/graphql/monitor_states.ts:
```diff
@@ -0,0 +1,59 @@
+        .set('kbn-xsrf', 'foo')
+        .send({ ...getMonitorStatesQuery });
+
+      expect(data).to.eql(monitorStates);
```
This test is failing in CI because the timestamps are still "2019" 🤔

justinkambic replied:
Yes I'm investigating now. Rather not good.

justinkambic replied:
Fixed in https://github.com/elastic/kibana/pull/40217/commits/31439818b61bbc910ecaf1cf5e6a9fad71cdb392 and https://github.com/elastic/kibana/pull/40217/commits/e43db48527e833cc01368b86f690ecf41073a78d.

elasticmachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pull-request/6454/)

justinkambic commented:
Backported to:
`7.3`/`7.3.0` https://github.com/elastic/kibana/commit/7fd9f7b0bb81e40f5a4af9983e7a1626a81c2460
  https://github.com/elastic/kibana/pull/40691
